beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -106,10 +109,9 @@ const REQUEST_TIMEOUT: u64 = 15;
 
 /// Returns the maximum bytes that can be sent across the RPC.
 pub fn max_rpc_size(fork_context: &ForkContext) -> usize {
-    if fork_context.fork_exists(ForkName::Merge) {
-        MAX_RPC_SIZE_POST_MERGE
-    } else {
-        MAX_RPC_SIZE
+    match fork_context.current_fork() {
+        ForkName::Merge => MAX_RPC_SIZE_POST_MERGE,
+        _ => MAX_RPC_SIZE,
     }
```

Comment:
I think it would be better to no use wildcard matching here and spell out each fork explicitly, even if it's just
```
    match fork_context.current_fork() {
        ForkName::Base | ForkName::Altair => MAX_RPC_SIZE,
        ForkName::Merge => MAX_RPC_SIZE_POST_MERGE,
    }
```
That way, when the next fork comes along, the compiler will call our attention to this line and we won't miss it if this changes in future forks.